weblate/lang/models.py

```diff
@@ -1118,7 +1119,7 @@ def list_plurals(self):
             yield {
                 "index": i,
                 "name": self.get_plural_name(i),
-                "examples": ", ".join(self.examples.get(i, [])),
+                "examples": format_html_join_comma("{}", self.examples.get(i, [])),
```

Comment:
Conversion to tuple is missing here?

Reply:
The format_html_join function spreads the elements of the lists(as in this case). We need a tuple when there is a string.